### DISCUSSION TITLE
2021-50: 奇妙的探险

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#309] [Announcements]
# 2021-50: 奇妙的探险

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2021-50/

**Comment by pkvov** [comment]
还是很不错的

**Comment by pymongo** [comment]
> 现在就是特别后悔，当年早该攒钱升级设备的，受了太多委屈。

我也后悔没早点用台式机哈哈，自从 mac 换成 amd 5900X 加上 lld/mold linker 之后，编译速度快了 6-7 倍率不止，还比 mac M1 快好几倍

我自己测了下，我这个 5900X 编译代码的速度还是比公司那些好几万一台的戴尔服务器略微快一些

CPU 性能越好，用 rust-analyzer 的体验就巨好，改一行代码瞬间编译完给出类型提示的反馈
